.github/workflows/nb2post.yaml

```diff
@@ -0,0 +1,41 @@
+name: Notebook to Blog Post
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '_notebooks/*.ipynb'
+
+jobs:
+  convert-notebook:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Copy Repository Contents
+      uses: actions/checkout@master
+
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.6'
+        architecture: 'x64'
+    
+      # These are the minimal steps I could come up with install nbdev in the actions runner
```

Comment:
@sgugger perhaps you would be interested in my install steps, I tried to make it as minimal as possible but these are the steps I found to make it work for me.  It still installs very very fast, though so no worries.